### PR TITLE
denonavr: Add DynamicEQ

### PR DESCRIPTION
**Pull request by MarBra**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#44930
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/denonavr.markdown [hunk 1]
```diff
@@ -119,4 +124,13 @@ A comprehensive list of telnet protocol commands is [also available](http://asse
 | `entity_id`            |       no | Name of entity to send command to. For example `media_player.marantz`|
 | `command`              |       no | Command to send to device, e.g.,  `/goform/formiPhoneAppDirect.xml?VSMONI2`|
 
+#### Service `denonavr.set_dynamic_eq`
+
+Enable or disable DynamicEQ setting.
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | Name of entity to send command to. For example `media_player.marantz`|
```
Comment (frenck) [inline source/_integrations/denonavr.markdown]: This is an entity service, meaning entity_id is optional.

File: source/_integrations/denonavr.markdown [hunk 2]
```diff
@@ -98,6 +98,11 @@ zone2:
   required: false
   default: false
   type: boolean
+update_audyssey:
+  description: Specifies if Audyssey settings should be updated. This can take up to 10 Seconds for some receivers.
+  required: false
+  default: false
+  type: boolean
```
Comment (frenck) [inline source/_integrations/denonavr.markdown:96]: This is part of the options flow now?
Additionally, an entity service was introduced to handle this option.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @MarBra 👋 

Could you be so kind to resolve the merge conflict and update the documentation to match the final PR that was merged into the Home Assistant Core?

Thanks! 👍 

../Frenck 🚂

File: source/_integrations/denonavr.markdown (hunk 2 above)
Comment (MarBra) [reply source/_integrations/denonavr.markdown:96]: Yes it is part of options flow to allow users to update Audyssey settings as part of the update cycle.
Additionally an entity service was added to allow the user to update the settings by them self. I have added the service to the docs.

File: source/_integrations/denonavr.markdown (hunk 1 above)
Comment (MarBra) [reply source/_integrations/denonavr.markdown]: Done 👍

**Comment by MarBra** [comment]
> Hi there @MarBra 👋
> 
> Could you be so kind to resolve the merge conflict and update the documentation to match the final PR that was merged into the Home Assistant Core?
> 
> Thanks! 👍
> 
> ../Frenck 🚂

Hi @frenck 
Thanks for review :) I've just updated the PR.

**Review by frenck — APPROVED** [review]
Thanks, @MarBra 👍 

../Frenck 🚂

**Pull request merged**